zebra/zebra_dplane.c

```diff
@@ -5764,6 +5764,21 @@ void dplane_provider_enqueue_out_ctx(struct zebra_dplane_provider *prov,
 				  memory_order_relaxed);
 }
 
+static struct zebra_dplane_ctx *
+dplane_provider_dequeue_out_ctx(struct zebra_dplane_provider *prov)
+{
+	struct zebra_dplane_ctx *ctx;
+
+	ctx = dplane_ctx_list_pop(&(prov->dp_ctx_out_list));
+	if (!ctx)
+		return NULL;
+
+	atomic_fetch_add_explicit(&(prov->dp_out_queued), -1,
```

Comment:
we have "fetch_sub" ?

Reply:
yeah, I don't really get why it even worked with "-1" : the type of the counter is uint32_t, so I'd have thought the negative number might have gotten turned into an unsigned?